e2e/_suites/fleet/ingest_manager_test.go

```diff
@@ -215,3 +223,31 @@ func InitializeIngestManagerTestSuite(ctx *godog.TestSuiteContext) {
 		}
 	})
 }
+
+func pullFreshImages(stackVersion, agentVersion, kibanaVersion string) error {
```

Comment:
Wdyt about moving an abstraction of the PullFreshImages method to the internal docker package?